test/istanbul-references.test.ts

```diff
@@ -59,7 +60,11 @@ const suites = await Promise.all(
 describe.each(suites)("$suite", async ({ tests }) => {
   describe.each(tests)("$name", (t) => {
     test.for(t.tests)("$name", async (testCase, ctx) => {
-      if (testCase.name?.includes("ignore") || t.name.includes("ignore")) {
+      // TODO: Add support for ignoreClassMethods
```

Comment:
So close to 100%!

![image](https://github.com/user-attachments/assets/2347db50-032d-4701-85dc-ff9939ce1a18)